src/main/java/com/facebook/presto/s3/S3TableDescriptionSupplier.java

```diff
@@ -65,6 +67,10 @@
 
         ImmutableMap.Builder<SchemaTableName, S3Table> builder = ImmutableMap.builder();
         List<Bucket> listOfBuckets = this.accessObject.listBuckets();
+        Set<String> set = listOfBuckets.stream().map(Bucket::getName).map(String::toLowerCase).collect(Collectors.toSet());
+        if(set.size()<listOfBuckets.size()){
+            throw new IllegalArgumentException("Duplicate bucket name found");
```

Comment:
Somewhat misleading.  Please change to indicate that "two or more buckets have been found with the same name, but different case of some of the letters"

Reply:
But the exception will also be thrown for cases where the user declares a bucket with the exact same name, won't the error message "two or more buckets have been found with the same name, but different case of some of the letters" be misleading in that case ?